packages/cli/src/commands/validator/affiliation.ts

```diff
@@ -25,17 +23,19 @@ export default class ValidatorAffiliate extends BaseCommand {
 
   async run() {
     const res = this.parse(ValidatorAffiliate)
-    const contract = await Validators(this.web3, res.flags.from)
+
+    this.kit.defaultAccount = res.flags.from
+    const wrapper = await this.kit.contracts.getValidators()
```

Comment:
I notice that you've been using the contract name other places, but just wrapper here - is that due to a collision somewhere?

Reply:
+1. Would be nice to keep the convention of naming these variables after the contract. I see `validators` referring to a list of validators in a command below. Maybe rename that `validatorList`, `registeredValidators` or something like that?